cypress/e2e/finance/funds/add-expense-class-to-budget.cy.js

```diff
@@ -6,9 +6,7 @@ import testType from '../../../support/dictionary/testTypes';
 import devTeams from '../../../support/dictionary/devTeams';
 import NewExpenceClass from '../../../support/fragments/settings/finance/newExpenseClass';
 import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
-import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
 import TopMenu from '../../../support/fragments/topMenu';
-import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
 import NewFiscalYear from '../../../support/fragments/finance/fiscalYears/newFiscalYear';
```

Comment:
'NewFiscalYear' is defined but never used. Allowed unused vars must match /React/u